odk/odk.py

```diff
@@ -567,10 +570,16 @@ class OntologyProject(JsonSchemaMixin):
     """if true add a gzipped version of the main artefact"""
     
     release_artefacts : List[str] = field(default_factory=lambda: ['full', 'base'])
-    """A list of release artefacts you wish to be exported."""
+    """A list of release artefacts you wish to be exported. Supported: base, full, baselite, simple, non-classified, 
+    simple-non-classified, basic."""
     
     release_use_reasoner : bool = True
-    """If set to True, the reasoner will be used during the release process."""
+    """If set to True, the reasoner will be used during the release process. The reasoner is used for three operations:
+    reason (the classification/subclassOf hierarchy computaton); materialize (the materialisation of simple existential/
+    object property restrictions); reduce (the removal of redundant subclassOf axioms)."""
```

Comment:
From @dosumis:

> More minor concern - language used to describe the changes in doc above and in the makefile options:
    release_use_reasoner : bool = True
    """If set to True, the reasoner will be used during the release process."""
> Use the reasoner is ambiguous. There are many potential uses of a reasoner. I think this means - assert inferred subClassOf axioms between named classes. If so, text associated with the option should make that clear (probably var name should too)

@matentzn has updated the description to be clear that using the reasoner means: reason, materialize and reduce.